manifests/e/Eltima/SerialPortMonitor/9.5.589/Eltima.SerialPortMonitor.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Eltima.SerialPortMonitor
+PackageVersion: 9.5.589
```

```suggestion
PackageVersion: "9.5"
```

Please update file path to match this change.